telemetry_subscribers/src/lib.rs

```diff
@@ -64,14 +68,49 @@ type ChromeGuard = tracing_chrome::FlushGuard;
 #[cfg(not(feature = "chrome"))]
 type ChromeGuard = ();
 
-pub struct TelemetryGuards(WorkerGuard, Option<ChromeGuard>);
+pub struct TelemetryGuards(Vec<WorkerGuard>, Option<ChromeGuard>);
+
+fn get_output(config: &TelemetryConfig) -> (BoxMakeWriter, Vec<WorkerGuard>) {
+    let cur_dir = std::env::current_dir().ok().unwrap().join("logs");
+    std::fs::create_dir_all(&cur_dir).ok().unwrap();
+    let (nb_stdout, nb_stdout_guard) = tracing_appender::non_blocking(std::io::stdout());
+    let (nb_stderr, nb_stderr_guard) = tracing_appender::non_blocking(std::io::stderr());
+
+    // default setting:
+    // (after filtering, controlled by env_filter)
+    // TRACE -> stdout
+    // DEBUG -> stdout
+    // INFO  -> stdout, stdout logfile (if enabled)
+    // WARN  -> stdout, stdout logfile (if enabled)
+    // ERROR -> stderr, stderr logfile (if enabled)
+
+    let mk_writer = nb_stdout
+        .with_max_level(Level::TRACE)
```

Comment:
1. Can we move this to later in the path where `config.log_file` is not set?
2. I thought in the description you said that the log levels can be chosen by environment filter, but here we are prescribing which log level... so I don't understand.  This seems to fix the level.  Unless it's a default, but I don't see anything here to interact with env_filter.
3. Trace should not be a default.  it is really really noisy.  The default should be INFO, but if you use env_filter it already defaults to that.